src/OrchardCore.Modules/OrchardCore.Search.Elasticsearch/Views/Admin/Index.cshtml

```diff
@@ -20,7 +20,8 @@
                     <input asp-for="Options.Search" class="form-control" placeholder="@T["Search"]" type="search" autofocus />
                 </div>
                 <div class="btn-group float-end">
-                    <a asp-route-action="Edit" class="btn btn-sm btn-secondary float-end create" role="button">@T["Add Index"]</a>
+                    <a asp-route-action="SyncSettings" title="@T["Synchronize content index settings with Lucene."]" class="btn btn-sm btn-secondary sync" role="button" data-url-af="RemoveUrl UnsafeUrl" data-title=@T["Synchronize Content Index Settings"] data-message=@T["Are you sure you want to synchronize all Elasticsearch content index settings with Lucene ones?"]>@T["Synchronize Settings"]</a>
```

Comment:
Two notes about styling:

1. The magnifying glass icon is now overlapping the "Search" text.
2. The two buttons are not separated.

![image](https://user-images.githubusercontent.com/1976647/190928948-4e494c85-d7dc-4855-a4ec-0c5cb9d6f690.png)

Reply:
Fixed